app/build.gradle

```diff
@@ -170,4 +170,6 @@ dependencies {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
     implementation files('libs/YouTubeAndroidPlayerApi.jar')
+    implementation 'com.google.android.exoplayer:exoplayer:2.9.6'
```

Comment:
I needed to add these dependencies because the project wasn't able to build :|. EmbeddedView wasn't able to find an ExoPlayer dependency